napari/_qt/_qapp_model/_tests/test_qapp_model_menus.py

```diff
@@ -21,5 +21,10 @@ def test_build_qmodel_menu(qtbot, menu_id):
         init_qactions()
         menu = build_qmodel_menu(menu_id)
         qtbot.addWidget(menu)
+
+        # empty menu not registering actions, for example help menu
+        if not menu.actions():
+            return
```

Comment:
Since we do `init_qactions` above, all actions should be registered - why does the help menu not register actions? What about the `Q_Help_Actions` ?

Reply:
Somehow the qactions don't show up as actions, I tested adding a fake action (I think I added a submenu) in the app action side then it magically worked

Reply:
I probably understand this wrong. going to debug on this: interestingly I noticed that the menu is there in the first iteration of the test loop (all 8 menus registered properly) when the menu_id in the test case is napari/view, however, somehow the help menu disappears from the second loop and on when running other test cases

Reply:
fixed